src/Sylius/Component/Grid/Filter/EntityFilter.php

```diff
@@ -29,8 +29,16 @@ public function apply(DataSourceInterface $dataSource, $name, $data, array $opti
         }
 
         $field = isset($options['field']) ? $options['field'] : $name;
+        $fields = isset($options['fields']) ? $options['fields'] : [$field];
```

Comment:
Unexpected? If both are set, this implementation uses only fields as it is more complex setting and ignores a field.